Assets/Scripts/Models/World.cs

```diff
@@ -159,6 +162,12 @@ public void DeleteRoom(Room r)
         r.ReturnTilesToOutsideRoom();
     }
 
+    public void AddEventListeners()
+    {
+        TimeManager.Instance.EveryFrameUnpaused += TickEveryFrame;
+        TimeManager.Instance.FixedFrequencyUnpaused += TickFixedFrequency;
```

Comment:
You should probably un-register this when the world get GC, we plan to have N worlds in a game session.